app/messages/messages/message.rb

```diff
@@ -27,8 +42,16 @@ def build_children(object, field)
       end
     end
 
-    private
```

Comment:
Was this intentional?

Reply:
Yes. I wanted to add some comments to the method and they don't show if the method is private. The method also has some duplication with some work Harrys has done so we will probably abstract the method out so it will need to be public anyway.

Reply:
OK great thanks.